browser/src/control/Control.ContextToolbar.ts

```diff
@@ -0,0 +1,296 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * L.control.ContextToolbar - definition of context toolbar
+ * This toolbar should appear over selection when related selection is done using mouse
+ */
+
+class ContextToolbar {
+	container: HTMLElement;
+	builder: any;
```

Comment:
JSBuilder

Reply:
unfortunatly it tried using JSbuilder its not possible as we are reusing some of the notebookbar builder handlers.

Reply:
I don't understand. `builder: JSBuilder` should work :) I want using a type.
If some properties/methods are missing in the interface - please add them to the definition.